Angular 15

**Describe the bug**
Currently cannot install apollo-angular inside of an angular 15 repository.

**Additional context**
I recently upgraded my repository to angular 15 and cannot install apollo-angular due to dependency conflict.

Hello
Me too cannot install apollo-angular v.4.1.1 ( I'm using angular v15.0.0 / node v18.12.1 / npm v8.19.2), apparently is not compatible with angular v15 yet.

rm__v2 % ng add apollo-angular
ℹ Using package manager: npm
⚠ Unable to find compatible package. Using 'latest' tag.
⚠ Package has unmet peer dependencies. Adding the package may not succeed.

The package [apollo-angular@4.1.1](mailto:apollo-angular@4.1.1) will be installed and executed.
Would you like to proceed? Yes
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: rm--v2@0.0.0
npm ERR! Found: @angular/core@15.0.0
npm ERR! node_modules/@angular/core
npm ERR! @angular/core@"^15.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/core@"^14.0.0" from [apollo-angular@4.1.1](mailto:apollo-angular@4.1.1)
npm ERR! node_modules/apollo-angular
npm ERR! apollo-angular@"4.1.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR! /Users/.npm/_logs/2022-11-19T15_09_05_281Z-debug-0.log
✖ Packages installation failed, see above.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -88,7 +88,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        angular_version: [14]
+        angular_version: [14, 15]
         graphql_version: [15, 16]
     steps:
       - name: Use Node.js 16

--- a/packages/apollo-angular/package.json
+++ b/packages/apollo-angular/package.json
@@ -36,7 +36,7 @@
     "deploy:next": "yarn build && npm publish build --tag next"
   },
   "peerDependencies": {
-    "@angular/core": "^14.0.0",
+    "@angular/core": "^14.0.0 || ^15.0.0",
     "@apollo/client": "^3.0.0",
     "graphql": "^15.0.0 || ^16.0.0",
     "rxjs": "^6.0.0 || ^7.0.0"

